modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -99,7 +106,7 @@ cp bootstrap.sh bootstrap-sles15.sh
 ----
 . Open [path]``bootstrap-sles12.sh`` for modification.
 Scroll down until you can see the text shown below.
-If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign (``#``) at the beginning of the line.
+If ``exit 1`` exists in the file, comment it out by typing a hash or pound sign (&#35;) at the beginning of the line.
```

Comment:
elsewhere you removed it completely, iirc

Reply:
Yes, but Julio complained :wink: